expo/gbe/models.py

```diff
@@ -303,17 +303,18 @@ def get_shows(self):
                  for act in acts if act.accepted == 3 and act.is_current]
         return sum([list(s) for s in shows], [])
 
-    def get_schedule(self):
+    def get_schedule(self, conference=None):
```

Comment:
Actually...I was using the unparameterized schedule function in the personal schedule.  If you go to /gbe - and switch to historical view, it also shows you your old schedule.  I figured this was useful both for people booking this year's schedule and for folks like the class coordinator, @burlexpo  and I when we try to see what folks have done for us in the past.